wordpress-mobile/WordPress-iOS
# Domains Dashboard Card: Fix Domains presentation on iPad

## Description

iPad uses a details controller for presentation, requiring additional logic to support the presentation.

## Solution

Create a separate method to present DomainsDashboard on top of BlogDashboard directly. This way we can consistently present DomainsDashboard on both iPad and iPhone.

## To test:

1. Launch Jetpack on iPad
2. Log in to free WP.com website without a domain
3. Enable `Domains Dashboard Card` feature flag
4. Refresh home
5. Select a card
6. Domains Dashboard View should appear

## Regression Notes
1. Potential unintended areas of impact

Presentation breaking on iPhone

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual Testing

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [x ] I have considered adding unit tests for my changes.


## Video

https://user-images.githubusercontent.com/4062343/231693530-f27b5ad7-3869-4508-8425-6d1606e3ce11.mp4


https://user-images.githubusercontent.com/4062343/231693539-b65748d9-1247-44bf-99dd-744046185ad4.mp4
